src/PrestaShopBundle/Resources/config/services/adapter/product/_seo.yml

```diff
@@ -5,6 +5,6 @@ services:
   prestashop.adapter.product.options.redirect_target_provider:
     class: PrestaShop\PrestaShop\Adapter\Product\Options\RedirectTargetProvider
     arguments:
-      - '@prestashop.adapter.product.repository.product_preview_repository'
```

Comment:
Don't forget to clean `product_preview_repository` service definition as well

Reply:
ok so I can close the comment